images/macos/helpers/Xcode.Installer.psm1

```diff
@@ -30,7 +31,13 @@ function Invoke-DownloadXcodeArchive {
 
     # TO-DO: Consider replacing of xcversion with own implementation
     Write-Host "Downloading Xcode $resolvedVersion"
-    Invoke-ValidateCommand "xcversion install '$resolvedVersion' --no-install"
+    Invoke-ValidateCommand "xcversion install '$resolvedVersion' --no-install" | out-host
+    $fixedResolvedVersion = "$resolvedVersion" -replace " ", "_"
```

Comment:
`$fixedResolvedVersion` is not clear variable name.
May be `$xcodeXipName`?

Reply:
`$xcodeXip` -> `$xcodeXipFile`

Reply:
`$tempXipDir` -> `$tempXipDirectory`

Reply:
Done.